DigitalLearningSolutions.Data/Services/UserService.cs

```diff
@@ -204,15 +207,15 @@ public List<CentreUserDetails> GetUserCentres(AdminUser? adminUser, List<Delegat
 
             if (verifiedDelegateUsers.Count != 0)
             {
-                var delegateIds = verifiedDelegateUsers.Select(d => d.Id).ToArray();
-                userDataService.UpdateDelegateUsers(
+                userDataService.UpdateUser(
                     myAccountDetailsData.FirstName,
                     myAccountDetailsData.Surname,
                     myAccountDetailsData.Email,
                     myAccountDetailsData.ProfileImage,
                     myAccountDetailsData.ProfessionalRegistrationNumber,
                     myAccountDetailsData.HasBeenPromptedForPrn,
-                    delegateIds
+                    centreAnswersData.JobGroupId,
+                    1 // TODO This needs correcting to the correct UserId for the delegate record.

```

Comment:
Can you stick a ticket reference on this? 887 unless you see another ticket which you're sure should cover this.

Reply:
Same with other todos